htdocs/mri/jiv/get_file.php

```diff
@@ -92,58 +132,68 @@
     exit(4);
 }
 
+$PathPrefix = '';
 switch($FileExt) {
 case 'mnc':
-    $FullPath         = $mincPath . '/' . $File;
+    $PathPrefix       = $mincPath;
```

Comment:
Why are these all changed instead of just switching realpath to truepath and verifying that it fixes the security issues (and works..)?